src/initial-state-for-ssr.js

```diff
@@ -0,0 +1,11 @@
+import createMatcher from './create-matcher';
+
+export default ({ url, query, routes, history }) =>
+  Object.assign({},
```

Comment:
No versions of IE support `Object.assign`, and it doesn't appear to be included in `es5-shim` either.  So we'll either need to call it out that you need an `assign` polyfill, don't support IE (not really an option for us), or pull in `_.assign`.  I vote for the last option.